package.json

```
@@ -15,7 +15,7 @@
   "scripts": {
     "start": "craco start",
     "build": "craco build",
-    "test": "craco test",
+    "test": "craco test --coverage --testResultsProcessor=jest-sonar-reporter",
```

I don't think the `coverage` flag should be added to the bare `test` script: it slows down tests, and there is already a separate `test:cov` script for this.

Why is `jest-sonar-reporter` needed? According to [this answer](https://community.sonarsource.com/t/no-coverage-information-when-using-jest-and-jest-sonar-reporter/39028), it is a third-party module (that has been archived for 5 years), and is unnecessary.
```suggestion
    "test": "craco test",
```